config/initializers/webauthn.rb

```diff
@@ -0,0 +1,35 @@
+WebAuthn.configure do |config|
+  # This value needs to match `window.location.origin` evaluated by
+  # the User Agent during registration and authentication ceremonies.
+  config.origin = "http://www.login.service.localhost"
```

Comment:
A lot of these APIs need to know the hostname they're running on, which I guess makes sense but is also a potential source of bugs - we use `ENV["OIDC_IDP_ISSUER"] || ENV["REDIRECT_BASE_URL"]` for this in the OIDC initialiser.

Reply:
Yeah great point, also worth nothing this is one of the config bits that means this PR would not be ready for game-time, further polish would be required.

NB there's also the option to make the relaying party differ from the default.
The gem author leaves comments [here](https://github.com/alphagov/govuk-account-manager-prototype/pull/704/files/6b1c124feed46af49bb97208cd4077f3776fb545#diff-bc7d5d6cad7fa7aef1230014f824a1f497e64bf498e00f47dde966801a682b04R16-R18) and [here](https://github.com/alphagov/govuk-account-manager-prototype/pull/704/files/6b1c124feed46af49bb97208cd4077f3776fb545#diff-bc7d5d6cad7fa7aef1230014f824a1f497e64bf498e00f47dde966801a682b04R20-R23)